docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -289,6 +294,84 @@ spec:
 ** Go to the pipeline run with *Build* in the *Type* column and confirm that the `prefetch-dependencies` stage displays a green checkmark. This indicates that the build process successfully fetched all dependencies.
 * From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
 
+== Enabling prefetch builds for `rpm`
+
+Cachi2 has a dev-preview package manager capable of fetching `rpm` dependencies. This requires the use of a pair of `rpms.in.yaml` and `rpms.lock.yaml` files to be committed to your repository. You write a `rpms.in.yaml` file and the link:https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#what-is-this[rpm-lockfile-prototype] CLI tool resolves that to produce a `rpms.lock.yaml` file. Cachi2 fetches those specific rpms and enables your build to install them without network access.
+
+.Prerequisites
+* You have an up-to-date installation of link:https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#installation[rpm-lockfile-prototype].
+
+.Procedure
+
+To prefetch dependencies for a component build, complete the following steps:
+
+. Create a `rpms.in.yaml` file in your git repository, with the following contents:
+
++
+[source,yaml]
+----
+packages: [nethack] <1>
+contentOrigin:
+  repofiles: ["./fedora.repo"] <2>
+----
+<1> The `*packages*` list is the list of packages you want to install in your Container. Only include first-order dependencies here, i.e. the packages you want to install.
```

Comment:
I'm not sure if readers will understand "first-order dependencies." Maybe a note like this would be sufficient: "You don't have to declare transitive dependencies here, the `rpm-lockfile-prototype` tool will resolve them for you."